### PR TITLE
Bugfix markdown support for intro

### PR DESCRIPTION
**Pull request by maciejSamerdak**
Adds markdown blocks support for intro generation

### CODE REVIEW

File: blog/models.py [hunk 1]
```diff
@@ -156,34 +156,39 @@ def get_header_id(self, title: str) -> int:
...
     @staticmethod
-    def _extract_paragraph_text_from_paragraph_block(paragraphs: cycle, text: str) -> str:
+    def _extract_paragraph_text_from_block(blocks: cycle, text: str) -> str:
         space_between_texts = " "
-        next_text = strip_tags(next(paragraphs).value.source)
+        next_block = next(blocks)
+        if next_block.block.name == ArticleBodyBlockNames.PARAGRAPH.value:
+            source_text = next_block.value.source
+        else:
+            source_text = next_block.value
```
Comment (kbeker) [inline blog/models.py:189]: Did you check what happen if user start block with something like `###` from markdown?

Comment (kbeker) [reply blog/models.py:189]: I think you should check if it is mardkown block and extract text from markdown signs

**Review by kbeker — CHANGES_REQUESTED** [review]
Please add test which will test when block has markdown symbols

File: blog/models.py (hunk 1 above)
Comment (maciejSamerdak) [reply blog/models.py]: Done

**Review by kbeker — APPROVED** [review]
Great!

**Pull request merged**